symfony/3-apache/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/continuouspipe/php-apache:7.0_v1
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -yq install \
+    acl \
```

Comment:
Is it to have `setfacl` ? If yes, wouldn't in makes sense to move it to the Ubuntu base image actually, as we'll have this problem often I believe!